IBAnimatableApp/RefreshControlTableViewViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Created by Eric Marchand on 31/03/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+import IBAnimatable
+
+public class RefreshControlTableViewViewController: AnimatableViewController {
+
+  @IBOutlet fileprivate weak var tableView: AnimatableTableView!
+  fileprivate let rows = ["Did you try to pull?",
+                          "Nice control, isn't it?",
+                          "Fully customisable in interface builder",
+                          "Amazing!"]
+
+  override public func viewDidLoad() {
+    super.viewDidLoad()
+
+    // Install action on refresh
+    let refreshControl: UIRefreshControl?
+    if #available(iOS 10.0, *) {
+      refreshControl = tableView.refreshControl
+    } else {
+      refreshControl = UIRefreshControl()
+      tableView.addSubview(refreshControl!)
+      tableView.configureRefreshController()
```

Comment:
I made a comment on https://github.com/IBAnimatable/IBAnimatable/pull/429/files#diff-f56d938957f5c1840f24beee6993086fR43

If we can't do it automatically, we may need to put this code to `AnimatableTableView` as a comment for pre-iOS 10.